### PR TITLE
Bump minimum supported watchOS version

### PR DESCRIPTION
**Pull request by sebastian-qm**
Hi!

My app doesn't compile with Xcode 15 (currently in beta). The problem is caused by the opentelemetry-swift package. I got the following errors in Xcode build log:

> Build target OpenTelemetryProtocolExporterCommon
> The package product 'Logging' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> The package product 'SwiftProtobuf' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> 
> Build target OpenTelemetryProtocolExporterGrpc
> The package product 'Logging' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> 
> The package product 'SwiftProtobuf' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> The package product 'GRPC' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> 
> Build target OpenTelemetryProtocolExporterHttp
> The package product 'Logging' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> The package product 'SwiftProtobuf' requires minimum platform version 5.0 for the watchOS platform, but this target supports 4.0
> 

I could fix the issue by forking opentelemetry-swift, bumping the watchOS version in your Package@swift5.6.swift and making my app depend on the fork.

### CODE REVIEW

**Comment by bryce-b** [comment]
We discussed this change and decided that supporting the Xcode Beta is a concern in an existing package manifest, which could break projects dependent on that manifest using the GA version of Xcode. Can you create a new package manifest for the beta xcode? Name it `Package@swift-5.9.swift`

**Comment by bryce-b** [comment]
I was looking at the support versions and Apple lists and xcode 15 is suppose to support watchOS 4.0 : 
https://developer.apple.com/support/xcode/

**Comment by sebastian-qm** [comment]
Ah, ok, got it. That would be `v10_15` (Catalina), right?

**Comment by nachoBonafonte** [comment]
We will also need you to sign the EasyCLA, we cannot merge if not signed

**Comment by sebastian-qm** [comment]
I'm in contact with my manager about the CLA. I think we can work it out next week. 

Thank you guys for your quick reviews! 👍

File: Package@swift-5.9.swift [hunk 1]
```diff
@@ -0,0 +1,226 @@
+// swift-tools-version:5.6
```
Comment (bryce-b) [inline Package@swift-5.9.swift:1]: set to 5.9?

**Comment by bryce-b** [comment]
@sebastian-qm Any update on getting the CLA signed?

**Comment by sebastian-qm** [comment]
> @sebastian-qm Any update on getting the CLA signed?

@bryce-b No, not really, sorry. I was on vacation last week 😎

File: Package@swift-5.9.swift (hunk 1 above)
Comment (sebastian-qm) [reply Package@swift-5.9.swift:1]: yes, makes sense. I'll change it.

**Comment by sebastian-qm** [comment]
EasyCLA is signed now.

**Review by nachoBonafonte — APPROVED** [review]

**Comment by sebastian-qm** [comment]
Hi @vvydier ,
could you please review this PR?

**Pull request merged**